gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageIntegrationHelper.java

```diff
@@ -49,7 +49,7 @@
 public abstract class GoogleCloudStorageIntegrationHelper {
 
   // Prefix used for naming test buckets.
-  public static final String TEST_BUCKET_NAME_PREFIX = "gcsio-test";
+  public static final String TEST_BUCKET_NAME_PREFIX = "dataproc-gcs-gcsio";
```

Comment:
why changing this?

Reply:
Because now we are running integration test against gRPC too.